vumi/transports/smpp/config.py

```diff
@@ -37,6 +37,11 @@ class SmppTransportConfig(Transport.CONFIG_CLASS):
         'for matching submit_sm_resp and delivery report messages. Defaults '
         'to 1 week.',
         default=(60 * 60 * 24 * 7), static=True)
+    completed_multipart_info_expiry = ConfigInt(
+        'How long (in seconds) to keep multipart message info for completed '
+        'multipart messages around to avoid pending operations accidentally '
+        'recreating them without an expiry time. Defaults to 1 hour.',
```

Comment:
Shouldn't the pending operations always create them with an expiring time too? Seems a bit scary to have any paths that can leave things in Redis forever? Perhaps I am just misunderstanding this help text though.